syntax = "proto2";
package GateServer;

option cc_generic_services = true;

message Request {
    required int32 flag = 1;
    optional string message = 2;
};

message Response {
    required int32 status = 1;
};

service UpStreamService {
    rpc Send(Request) returns (Response);
    rpc FirstSend(Request) returns (Response);
};